docs/UserGuide.md

```diff
@@ -60,12 +60,14 @@ Nuudle is a **desktop app for managing patient records and clinic appointments,
 
 ### Viewing help : `help`
 
-Shows a message explaining how to access the help page.
+Shows a message explaining how to access Nuudle's help page.
```

Comment:
would `help page of Nuudle` be better? or just another way to avoid not using `'s`

Reply:
Ok, will make the change! Thanks! 👍